packages/storybook/stories/INTERNAL/CallReadiness/DomainPermissions/DomainPermissions.stories.tsx

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Modal, PrimaryButton, Stack } from '@fluentui/react';
+import { _DomainPermissions } from '@internal/react-components';
+import { Canvas, Description, Heading, Props, Title } from '@storybook/addon-docs';
+import { Meta } from '@storybook/react/types-6-0';
+import React, { useState } from 'react';
+import { useLocale } from '../../../../../react-components/src/localization';
+import { SingleLineBetaBanner } from '../../../BetaBanners/SingleLineBetaBanner';
+import { COMPONENT_FOLDER_PREFIX } from '../../../constants';
+
+const DomainPermissionsStory = (): JSX.Element => {
+  const [modalOpen, setModalOpen] = useState<boolean>(false);
+  const locale = useLocale().strings.DomainPermissions;
+  return (
+    <Stack>
+      <Stack>
+        <Title>DomainPermissions Component</Title>
+        <SingleLineBetaBanner />
+        <Description>
+          Component to display information to the end user when their device permissions are not set appropriately
+        </Description>
+        <Heading>Example DevicePermissions</Heading>
+        <Heading>DomainPermissions Props</Heading>
+        <Props of={_DomainPermissions} />
+      </Stack>
+      <Canvas>
+        <PrimaryButton onClick={() => setModalOpen(true)}>Open Domain Permissions</PrimaryButton>
+        <Modal isOpen={modalOpen} onDismiss={() => setModalOpen(false)}>
+          <_DomainPermissions
+            appName={'Contoso app'}
+            onGetTroubleShooting={() => {
+              alert('clicked trouble shooting');
+            }}
+            strings={locale}
+          />
+        </Modal>
+      </Canvas>
+    </Stack>
+  );
+};
+
+// This must be the only named export from this module, and must be named to match the storybook path suffix.
+// This ensures that storybook hoists the story instead of creating a folder with a single entry.
+export const DomainPermissions = DomainPermissionsStory.bind({});
+
+export default {
+  id: `${COMPONENT_FOLDER_PREFIX}-internal-domain-permissions`,
+  title: `${COMPONENT_FOLDER_PREFIX}/Internal/CallReadiness/DomainPermissions/Domain Permissions`,
```

Comment:
Something's not quite worked here, there's 4 entries for domain permissions?
![image](https://user-images.githubusercontent.com/2684369/189454769-3f2c5bcb-dc3c-45ae-a5bc-6c8d10e3e7ab.png)